feat(legacy): disable public radio page and redirect to login (#2903)
libretime/libretime: docs/user-manual/settings.md

week that you wish to start your station's weekly schedule on, which defaults
to Sunday.

:::note

> The **Station Timezone** setting can not be modified here and is set in the master
> config file. Please consult
> [this admin manual section](../admin-manual/configuration/#general) on how to
> change it.

Reviewer: ```suggestion
The **Station Timezone** setting can not be modified on this page. It is set in the [configuration file](../admin-manual/configuration/#general).
```

Somewhat unrelated to this PR, but we'll let it slip :wink: .

Author: Sorry, i have a habit to update the page i am working on, if i see stuff is outdated. Can't really help it :-)

Reviewer: Great that you did!

[_Exporting the schedule_](./playout-history.md) chapter, in the
_Advanced Configuration_ section of this book).

:::note

> The **Allowed CORS URLs** you can still see in this screenshot was moved to the
> master config file. Please consult
> [this admin manual section](../admin-manual/configuration/#general) on how to
> change it.

Reviewer: ```suggestion
The **Allowed CORS URLs** you can still see in this screenshot was moved to the [configuration file](../admin-manual/configuration/#general).
```
